content/accessibility/accessibility-at-github.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Accessibility at GitHub
+---
+
+At GitHub our vision is to be the home for *all* developers. To be inclusive, that means we need to consider accessibility at our core of how we design. 
+
+15% of the world's population has a disability (estimated by the [World Health Organization based on 2010 data](https://www.who.int/disabilities/world_report/2011/report/en/)), and that doesn't take into consideration those under temporary or situational disabilities. 
+
+In many cases, devices that have been designed with accessibility in mind for a portion of the population ends up benefiting everyone. Take the [OXO brand's Good Grips line of products](https://www.oxo.com/blog/behind-the-scenes/behind-design-oxos-iconic-good-grips-handles/); although they were developed for people who have arthritis, most people found them easier to use than other product lines.
+
+If we can include accessibility at the beginning of the design stage and incorporate it into our process, we no longer have to treat it as the  complex add-on that looms in the corner as our designs come closer to creation. We can see it as part of our ingrained design process that seeks to include everyone, of all abilities, and allow them to achieve more.
+
+## What standards are we aiming for? 
+GitHub aims for [Web Content Accessibility Guidelines (WCAG) 2.1](https://www.w3.org/TR/WCAG21/) AA compliance. This includes all of WCAG 2.0 AA plus additional considerations. 
+
+Currently, [Section 508](https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-ict-refresh/overview-of-the-final-rule) required by law in the United States follows WCAG 2.0 AA guidelines for compliance. 
+
+## Who should we consider when designing? 
+There are a whole host of disabilities to consider, though the majority will fall into one of these categories: 
+- **Visual:** Anything that deals with sight, or technology that helps someone to see. This includes people who have different types of sight like color blindness.
+- **Cognitive:** Concentration, memory, judgement, problem solving, logic skills. 
+- **Mobility:** Anything that affects movement in a body. 
+- **Hearing:** A spectrum of disabilities related to sound or audio.
+
+Each of these categories will have three categories themselves:
+- **Permanent:** disability does not go away.
+- **Temporary:** the mismatched human interaction will resolve in time.
```

Comment:
This is a strange sentence. Could we rephrase it more simply?

Reply:
Yep. I had it changed on Figma (and I missed adding this change to the code) like this: "Each of these disability categories can be divided into three further sub-categories:"

Does that sound better?

Reply:
yes 👍🏼